users/views.py

```diff
@@ -125,3 +132,32 @@ def friends(request, author_id):
                     {"author": author,
                      "followers": author.get_followers(),
                      "following": author.get_following()})
+
+def editProfile(request, author_id):
```

Comment:
there is no reason to ever GET /author/id/edit, so use the `@require_POST` decorator from `django.views.decorators.http`, and get rid of the `if request.method == "POST"`